server/helpers/verifyEmailMessage.js

```diff
@@ -16,7 +16,7 @@ const verifyEmailMessage = (token) => {
     `<div style="height: 20em, background-color: #E6FFED; 
       border: 1px solid black; padding: 0.5em;">
         <p>
-          <a href="http://localhost:3001/api/test/verify?token=${token}">
+          <a href="http://localhost:3001/api/v1/users/verify?token=${token}">
```

Comment:
I believe that building out the URL this way would pose a problem later on. Please meet with @Darthrighteous He seems to have a similar acceptable solution for this.

Reply:
@andela-moe thanks. I'll get on this.